### DISCUSSION TITLE
Is it possible to put my script in an apk file?

### DISCUSSION THREAD

**Discussion by JieGeLovesDengDuaLang** [vfsfitvnm/frida-il2cpp-bridge#521] [Q&A] [answered]
I want to put an script (maybe also frida environment) in an apk file so that the script can run when i open the app without root. Is it possible? If so, what do i have to do with my script?

**Comment by 946923759** [comment, marked as answer]
This is possible with https://github.com/sensepost/objection

Simply run `objection patchapk -a arm64-v8a -s YOUR_APK_HERE.apk -c objection_config.json -l _.js` (where `_.js` is your script name)
